study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/EmailBuilder.java

```diff
@@ -100,8 +100,9 @@ private EmailInfo createSendGridEmail(Config emailCfg) {
         }
         String subject = emailCfg.getString("subject");
         String name = emailCfg.getString("name");
+        boolean isDynamic = emailCfg.getBoolean("isDynamic");
```

Comment:
wondered if we wanted to do:
boolean isDynamic = false;
if (emailCfg.hasPath("isDynamic")) {
    isDynamic = emailCfg.getBoolean("isDynamic");
}

This way we don't need to update all existing studies email configs with isDefault = false, but may be its little more cleaner having the property set explicitly rather than defaulting.

Reply:
I think I will have preferred defaulting, but since you have already made the changes let's just keep it the way you have it.